docs/content/preview/yugabyte-voyager/reference/assess-migration-bulk.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Bulk assess migration
```

Comment:
Title should be - `assess migration bulk` since the command is: `yb-voyager assess-migration-bulk --args...`

Like we have `export schema` `analyze schema`